pkg/kubectl/cmd/set/set_serviceaccount_test.go

```diff
@@ -93,109 +86,6 @@ func TestServiceAccountLocal(t *testing.T) {
 	}
 }
 
-func TestServiceAccountRemote(t *testing.T) {
```

Comment:
can not get info.VersionedObject in this cases. @liggitt do you know this?

now just remove this for test others, will retrieve it~